science/PothosLiquidDSP/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+
+platforms           darwin macosx
+categories          science
+license             Boost-1
+maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintainer
+
+description         Pothos toolkit for liquid DSP processing cores
+long_description    This toolkit wraps Pothos blocks around the Liquid \
+    DSP library. Blocks are generated from the liquid DSP include header \
+    and a file describing the available functions and parameters. \
+
+github.setup        pothosware PothosLiquidDSP 98b157a0d064fed19df05179172376ea15b0d31b
+version             0.0.0-git-20190409
+checksums           rmd160  fc9b1e4bc33594f917f5bb16babf4e42c7a36e9e \
+                    sha256  b0aab1b61b838f4e7b764b0e32b76a2e9f961179c1ff58454043a80b6dd3bbc2 \
+                    size    37459
+revision            0
+
+set python_version  36
+
+depends_build-append \
+    port:py${python_version}-ply \
+    port:py${python_version}-yaml \
+    port:py${python_version}-mako
+
+depends_lib-append \
+    port:PothosCore \
+    port:liquid-dsp
```

Comment:
Needs the following to ensure the correct Python is used:
```
set PythonVersionWithDot [join [split ${python_version} ""] "."]

configure.args-append \
    -DPYTHON_EXECUTABLE=${frameworks_dir}/Python.framework/Versions/${PythonVersionWithDot}/bin/python${PythonVersionWithDot} \
    -DPYTHON_INCLUDE_DIR=${frameworks_dir}/Python.framework/Versions/${PythonVersionWithDot}/Headers \
    -DPYTHON_LIBRARY=${frameworks_dir}/Python.framework/Versions/${PythonVersionWithDot}/Python
```

Reply:
done, thank you.